python/ray/data/dataset_pipeline.py

```diff
@@ -1071,7 +1073,7 @@ def iter_tf_batches(
         """Call
         :py:meth:`Dataset.iter_tf_batches <ray.data.Dataset.iter_tf_batches>`
         over the stream of output batches from the pipeline."""
-        return Dataset.iter_tf_batches(
+        return DatasetIterator.iter_tf_batches(
```

Comment:
`DataIterator` -> `DatasetIterator` because the dependency PR isn't cherrypicked.